playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.scss

```diff
@@ -183,34 +201,62 @@
 
     // Dark Mode Placeholder 
```

Comment:
For some reason, the dark mode background transition is slower on the bar than it is in the text area. Let's sync those or perhaps remove the transition on the bar.